src/main/java/ru/kovalchuk/Helper.java

```diff
@@ -1,66 +1,158 @@
 package ru.kovalchuk;
 
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+
 public class Helper {
-    static boolean check_exist_task(String[][] list_task) {
-        if (list_task[1][0].equals("")){
+    private static final String inProgress = ". [ ] "; // задача не выполнена
+    private static final String done = ". [v] "; // задача выполнена
+
+    static boolean checkExistTask(List<Task> taskList) {
+        if (taskList.size() == 0){
             System.out.println("Не найдено ни одной задачи");
             return false;
         }
         return true;
     }
 
-    static void toggle_task(String in_progress, String done, String[][] list_task, Integer id_task) {
-        if (id_task > 0 & id_task < list_task.length)
-        {
-            if (list_task[id_task][1].equals(in_progress)) {
-                list_task[id_task][1] = done;
+    static void toggleTask(List<Task> taskList, String idTask) {
+        if (Helper.checkExistTask(taskList)) {
+            try {
+                int id = Integer.parseInt(idTask) - 1;
+                Task task = taskList.get(id);
+                task.setDone(!task.isDone());
+                printTaskInfo(id, task);
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("Задачи с таким id нет");
+            } catch (NumberFormatException e){
+                System.out.println("Необходимо ввести id задачи цифрами");
             }
-            else {
-                list_task[id_task][1] = in_progress;
+        }
+    }
+
+    static void printTasks(List<Task> taskList, boolean flagAll) {
+        if (Helper.checkExistTask(taskList)) {
+            if (!flagAll){
```

Comment:
Вот так можно будет избавиться от if-else здесь :) 
```suggestion
                IntStream.range(0, taskList.size())
                .filter(i -> flagAll || !taskList.get(i).isDone())
                .forEach(i -> printTaskInfo(i, taskList.get(i)));
```